File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_ViewModelCustomListRowFieldRow.cpp

#include "LowLevelFields_ViewModelCustomListRowFieldRow.h"
#include "LowLevelFields_ViewModelCustomListRowFieldRow.h"
#include <vector>
#include <string>

namespace widgetassertions
{
  std::string LowLevelFields_ViewModelCustomListRowFieldRow::getHeaderLabelText( ) 
  {
    return this->HeaderLabelText;
  }
  std::vector<std::string> LowLevelFields_ViewModelCustomListRowFieldRow::getAdditionalStrings( ) 
  {
    return this->AdditionalStrings;
  }
  std::string LowLevelFields_ViewModelCustomListRowFieldRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_ViewModelCustomListRowFieldRow.cs
/// <filename>
///     LowLevelFields_ViewModelCustomListRowFieldRow.cs
/// </filename>
using System.Collections.Generic;

namespace widgetassertions
{
    public class LowLevelFields_ViewModelCustomListRowFieldRow
    {
        private string HeaderLabelText;

        public virtual string getHeaderLabelText()
        {
            return this.HeaderLabelText;
        }

        private System.Collections.Generic.List<string> AdditionalStrings = new System.Collections.Generic.List<string>();

        public virtual System.Collections.Generic.List<string> getAdditionalStrings()
        {
            return this.AdditionalStrings;
        }

        private string RowHandle;

        public virtual string getRowHandle()
        {
            return this.RowHandle;
        }
    }
}
